fineract-loan/src/main/java/org/apache/fineract/portfolio/loanproduct/api/LoanProductsApiResourceSwagger.java

```diff
@@ -157,6 +157,8 @@ private PostLoanProductsRequest() {}
         public Long delinquencyBucketId;
         @Schema(example = "false")
         public Boolean enableInstallmentLevelDelinquency;
+        @Schema(example = "false")
+        public Boolean useDueRepaymentGlobalConfigs;
```

Comment:
@adamsaghy / @bharathc27 

This boolean attribute is to let the UI know If the products uses or not the global config values. This is because in the UI exists a checkbox to indicate this. When you are creating and select that checkbox the global values are sent to Fineract. Until this, It is ok

The problem exists when you edit a Loan Product, the UI can not to know If the values of these configurations are or not copied from the global config

This issue was reported by Peter. The UI has the logic to read the value, but If Fineract does not return any value for this, then the UI can not indicate if the configurations are or not inherit from the global configurations

Reply:
I still believe we should not have the which indicates whether it is coming from global configuration or not. I would simply leave these fields as optional and if user does not fill them, it is coming from global configurations. If user provide a value then that will be used instead of the one from global configurations. Additionally we can enhance further and return the values from global configurations and we can show it on the UI to inform the user if nothing is sent, those value will be used, but they should be different values and do not send it back to backend! 

What are your thoughts?